h2o-core/src/main/java/water/H2O.java

```diff
@@ -44,6 +44,18 @@
 final public class H2O {
   public static final String DEFAULT_JKS_PASS = "h2oh2o";
   public static final int H2O_DEFAULT_PORT = 54321;
+  public static final Map<Integer, String> GITHUB_DISCUSSIONS = createMap();
+  
+  static Map<Integer, String> createMap() {
+    String[] GHDiscussion = new String[]{"15512", "15513", "15514", "15515", "15516", "15517", "15518", "15519",
```

Comment:
nit: this could be also array of integer and map int  to int

Reply:
True.  Fixed.